### DISCUSSION TITLE
Unavailable for your site because your domain is not properly configured to support HTTPS

### DISCUSSION THREAD

**Discussion by minglethepringle** [community/community#38529] [Other Feature Feedback, Questions, & Ideas]
My repo is https://github.com/minglethepringle/minglethepringle.github.io, my domain is mingleli.com . I put in all four A records pointing to GitHub pages, that works because http://mingleli.com is active. However, the CNAME in GoDaddy doesn't seem to work, per the DNS checker.
![image](https://user-images.githubusercontent.com/16504328/200473875-20be6ea9-3e51-4883-9cf3-ce975d62cb72.png)
![image](https://user-images.githubusercontent.com/16504328/200473899-a5cc4e84-c174-497f-819a-b24b0dd11242.png)

What could be wrong? I don't think it's letting me enforce HTTPS because of this

**Comment by yoannchaudet** [comment]
It resolves on my side but you have bad records in there:

```
dig mingleli.com

; <<>> DiG 9.10.6 <<>> mingleli.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 62131
;; flags: qr rd ra; QUERY: 1, ANSWER: 6, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;mingleli.com.			IN	A

;; ANSWER SECTION:
mingleli.com.		600	IN	A	185.199.109.153
mingleli.com.		600	IN	A	185.199.110.153
mingleli.com.		600	IN	A	185.199.111.153
mingleli.com.		600	IN	A	3.33.152.147
mingleli.com.		600	IN	A	15.197.142.173
mingleli.com.		600	IN	A	185.199.108.153

;; Query time: 6 msec
;; SERVER: 127.0.0.1#53(127.0.0.1)
;; WHEN: Mon Nov 07 20:51:18 PST 2022
;; MSG SIZE  rcvd: 137
```

Respectively `3.33.152.147` and `15.197.142.173` should not be there.

**Reply by minglethepringle** [reply]
I know it resolves but those records are put in by GoDaddy
![image](https://user-images.githubusercontent.com/16504328/200478513-307ee228-44ef-441b-aab2-b1c76f1c519a.png)

I can't delete them.

So it turns out that instead of putting "mingleli.com" I put "www.mingleli.com" and everything became okay, and now when I type "mingleli.com" the domain resolves/redirects to "www.mingleli.com" which I can put up with.

**Reply by yoannchaudet** [reply]
Glade you resolved it! DNS providers sometimes inject A records on your behalf when you use their "HTTP redirects" feature.
